saged/download_recount3.R

```diff
@@ -10,12 +10,6 @@
 # recount3 from being installed. I recommend running this script from a base r installation 
 # rather than a conda env
 
-#install.packages("BiocManager", repos = "https://cran.r-project.org", dependencies=TRUE)
```

Comment:
Maybe you could leave these install commands in but wrap them in a check to detect if they're already installed - something like [this might work?](https://stackoverflow.com/a/19871106)

Not sure what the convention is for this in R, though - maybe it's fine to just assume users already have the packages installed.

Reply:
This script was originally meant to be part of a standalone package,  which is why I included the libraries used. Since I never ended up making the package, I'm fine with leaving all the installation up to Renv